src/client/dom-processor/client-dom-adapter.js

```diff
@@ -8,6 +8,7 @@ import { getProxyUrl } from '../utils/url';
 import { isIE9 } from '../utils/browser';
 import * as domUtils from '../utils/dom';
 import fastApply from '../utils/fast-apply';
+import { hasIsNotClosedFlag } from '../sandbox/node/document/writer';
```

Comment:
After discussing with @VasilyStrelyaev  we decide use the `unclosed` word.
Accordingly, need to rename in the `DocumentWriter`:
```js
const IS_NOT_CLOSED_PROPERTY = 'hammerhead|is-not-closed-element';
hasIsNotClosedFlag
_setIsNotClosedFlag
...
const UNCLOSED_ELEMENT_FLAG = 'hammerhead|unclosed-element-flag';
setUnclosedElementFlag
hasUnclosedElementFlag
```